config/settings.yml

```diff
@@ -559,9 +559,9 @@ va_mobile:
 
 hqva_mobile:
   url: "https://veteran.apps.va.gov"
-  mock: true
-  cert_path: config/certs/vetsgov-localhost.crt
-  key_path: config/certs/vetsgov-localhost.key
+  mock: false
+  key_path: /srv/vets-api/secret/health-quest.key
+  local_key_path: modules/health_quest/config/rsa/sandbox_rsa
```

Comment:
@ericboehs All environments other than dev environments(local or anything which returns true for Rails.env.development?) will use the key pair that's in the repository. All other environments including, staging and production will use what Ops provided us: `/srv/vets-api/secret/health-quest.key`

Reply:
Maybe name it `development_key_path`?